### ISSUE TITLE
change invocation name

### ISSUE THREAD

**Issue by Yahoo-Mike** [infinityofspace/jellyfin_alexa_skill#8]
### Please describe which problem should be solved

[comment]: <> (A short and precise description of what problem the feature is supposed to address.)

Currently the invocation name is set to "jellyfin player".  I get about a 30% failure rate when I test this in the developer console.  Sometimes Alexa thinks I'm saying "jellyfish player" (lol) or "jellyfi player" (???) or even "elephant player" (!?!).  I think it's because "jellyfin" is not an English word.

I get a 100% hit rate with "jelly fin".  

### Describe your idea for a solution

[comment]: <> (A short and precise description of how to solve the above problem..)

It would be good to change the invocation name to "jelly fin" or have the ability to override invocation name in skills.conf.

If we change the invocation word in interaction_model_en_US.json to "jelly fin" (two words), then interactions become simpler.  For example: "Alexa, ask Jellyfin to play music by Billy Holiday".

Alternatively, there could be a section in the skills.conf to override the invocation name from the interaction json.  For example:

```
[en-US]
invocation_name=jelly fin

#[de-DE]
#invocation_name=mein server

```
If you like one of these ideas, I can make the changes.

**Comment by infinityofspace** [comment]
Thanks for your idea suggestion. The idea for changing the invocation name is very useful.

For the implementation your second suggestion would be the better choice, because it offers the best possible customization possibility.

### Some concrete implementation ideas:

The default invocation name will thus remain the same "jellyfin player" for all languages.

At each start, the interaction model is retrieved in addition to the skill manifest and compared with the stored skill invocation name ([this would be a good starting point](https://github.com/infinityofspace/jellyfin_alexa_skill/blob/939cafd6d8438a063d29b8727b13d444457972a3/jellyfin_alexa_skill/main.py#L154)). If the invocation name in the skill config is empty, the default name "jellyfin player" is used for comparison. When the invocation name in the currently built skill differs, then the name is written to the interaction model at runtime and sent to the AWS servers for a new skill build.

The partitioning in the skill config according to the languages and use the language codes (en-US, de-DE, etc.) as sections (as you described it)  is a good idea, because future adjustments depending on the language will also be included in this part:
```
[en-US]
invocation_name=jelly fin

[de-DE]
invocation_name=jellyfin player
```

You are welcome to create a PR if you like or you can also share other suggestions for a concrete implementation design.

**Comment by infinityofspace** [comment]
Closed in PR #9
